### PR TITLE
[MFTF] Add ActionGroup for clicking edit link on customer grid

### LINKED ISSUE #29501: [Issue] [MFTF] Add ActionGroup for clicking edit link on customer grid
This issue is automatically created based on existing pull request: magento/magento2#29460: [MFTF] Add ActionGroup for clicking edit link on customer grid

---------
This PR add `AdminCustomerClickFirstRowEditLinkActionGroup` to clicking edit link for first row on customer grid

### PR DESCRIPTION
**Pull request by Usik2203**
This PR add `AdminCustomerClickFirstRowEditLinkActionGroup` to clicking edit link for first row on customer grid

### Resolved issues:
1. [x] resolves magento/magento2#29501: [MFTF] Add ActionGroup for clicking edit link on customer grid

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by sidolov** [comment]
@magento create issue

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

File: app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminCustomerClickFirstRowEditLinkActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminCustomerClickFirstRowEditLinkActionGroup">
```
Comment (eduard13) [inline app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminCustomerClickFirstRowEditLinkActionGroup.xml]:
```suggestion
    <actionGroup name="AdminClickFirstRowEditLinkOnCustomerGridActionGroup">
```

**Review by eduard13 — CHANGES_REQUESTED** [review]
Could you please rename the action group as suggested?
Thank you.

**Comment by Usik2203** [comment]
@magento run all tests

File: app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminCustomerClickFirstRowEditLinkActionGroup.xml (hunk 1 above)
Comment (Usik2203) [reply app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminCustomerClickFirstRowEditLinkActionGroup.xml]: **Done**

**Review by eduard13 — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-8043](https://jira.corp.magento.com/browse/ENGCOM-8043) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Pull request merged**